lib/data_hygiene/deleted_document_restorer.rb

```diff
@@ -0,0 +1,26 @@
+require "active_record"
```

Comment:
Surprised the linter doesn't shout about the missing newline 🤔 

```suggestion
require "active_record"

```

Reply:
I don't even need active record, I was toying with the idea of throwing its `RecordNotFound` error but that didn't feel right. Forgot to take it out but have done so now.